src/components/elements/storyTranslationAndCitation.tsx

```diff
@@ -88,18 +130,15 @@ export const StoryTranslationAndCitation = (props: any) => {
     return (
         <>
             <Typography variant="subtitle1" fontWeight="bold"> TRANSLATION </Typography>
-            {ConstructEnglishTranslationAuthor(story.translation_author, story.translation_source_manuscript_name, story.translation_source_manuscript_folio)}
+            {ConstructEnglishTranslationAuthor(story.translation_author, story.translation_source_manuscript_name, story.translation_source_manuscript_folio, story.translations)}
             {ConstructEnglishTranslation(story.english_translation)}
```

Comment:
I just tried deploying this on lab1 and it's breaking story.translations doesn't exist on your api, so it would need to get added first